src/vue/common/idle/IdleMessage.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <div class="idle-wrapper">
+    <drawer
+      @close-drawer="closeIdleBar"
```

Comment:
`closeIdleBar` doesn't close your bar(?). It will forward user to another page, so name it correctly, please